redbot/cogs/mutes/mutes.py

```diff
@@ -639,20 +674,24 @@ async def _check_for_mute_role(self, ctx: commands.Context) -> bool:
         This explains to the user whether or not mutes are setup correctly for
         automatic unmutes.
         """
+        command_1 = f"{ctx.clean_prefix}muteset role"
+        command_2 = f"{ctx.clean_prefix}muteset makerole"
+        msg = _(
+            "This server does not have a mute role setup, "
+            " You can setup a mute role with `{command_1}` or"
+            "`{command_2}` if you just want a basic role created setup.\n\n"
+        ).format(command_1=command_1, command_2=command_2)
         mute_role_id = await self.config.guild(ctx.guild).mute_role()
         mute_role = ctx.guild.get_role(mute_role_id)
         sent_instructions = await self.config.guild(ctx.guild).sent_instructions()
+        force_role_mutes = await self.config.force_role_mutes()
+        if force_role_mutes and not mute_role:
+            await ctx.send(msg)
+            return False
         if mute_role or sent_instructions:
             return True
         else:
-            command_1 = f"{ctx.clean_prefix}muteset role"
-            command_2 = f"{ctx.clean_prefix}muteset makerole"
-            msg = _(
-                "This server does not have a mute role setup, "
-                "are you sure you want to continue with channel "
-                "overwrites? You can setup a mute role with `{command_1}` or"
-                "`{command_2}` if you just want a basic role created setup.)\n\n"
-            ).format(command_1=command_1, command_2=command_2)
+            msg += _("are you sure you want to continue with channel overwrites?")
```

Comment:
First letter uppercase and add space as last char so that the message appended later in this code has proper spacing.
```suggestion
            msg += _("Are you sure you want to continue with channel overwrites? ")
```